spring-cloud-starter-task-composedtaskrunner/README.adoc

```diff
@@ -123,6 +124,9 @@ definitions fails in the split then DDD and EEE will not fire.  For example if
 BBB fails then AAA and CCC will be marked successful and BBB will be marked a
 failure and DDD and EEE will not be launched.
 
+NOTE: Tasks that are used in a split should not set the their `ExitMessage`.   Setting the `ExitMessage` is only to be used
+with <<ctr-transitional-execution, transitions>>.
```

Comment:
Is there a way we could modify the CTR to ignore this?  I only ask because this means we need to implement tasks in a specific way to be orchestrated this way using the CTR which doesn't feel right.